eng/testing/performance/ios_scenarios.proj

```diff
@@ -19,6 +19,8 @@
     <LlvmPath Condition="'$(iOSLlvmBuild)' == 'True'">llvm</LlvmPath>
     <SymbolsPath>symbols</SymbolsPath>
     <SymbolsPath Condition="'$(iOSStripSymbols)' == 'True'">nosymbols</SymbolsPath>
+    <HybridGlobalizationPath></HybridGlobalizationPath>
+    <HybridGlobalizationPath Condition="'$(hybridGlobalization)' == 'True'"> HybridGlobalization</HybridGlobalizationPath>
```

Comment:
Nit: Would be good to align it with LlvmPath and SymbolsPath. Either use ` ` everywhere or remove it.

Reply:
That makes sense, I will switch to having all have a space in the property to help ensure consistent name spacing. This will make sure we are set for future additions to the names without having the potential for a bunch of extra white space throwing off the names.